build/jsdoc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "source": {
+    "include": [
+      "dev-api\\temp"
```

Comment:
This looks like winodws style path formats. Just verify that it works in linux too with WSL.

Reply:
Yupp. Tested with WSL and it works!

Moreover, we use path.join() method to set up file paths. This automatically checks for OS and set paths related to that. So we are all good in that case.